extensions/amp-carousel/0.1/build-dom.js

```diff
@@ -317,13 +319,16 @@ export function getPrevButtonTitle(element, options = {}) {
  * - Slides    : "Next item in carousel (X of Y)"
  *
  * @param {*} element
- * @param {{prefix: string, index: string, total:string}} param1
+ * @param {{prefix: string, index?: string, total?:string}} param1
  * @return {string}
  */
 function getButtonTitle(element, {index, prefix, total}) {
   if (isScrollable(element)) {
     return prefix;
   }
+  // Slides carousel should always have index/total provided.
+  devAssert(index);
+  devAssert(total);
```

Comment:
What is this meant to check? That the strings are non-empty?

Reply:
Yes